### ISSUE TITLE
Invalid version “6.3-201110161418”: non-numeric “3-201110161418”

### ISSUE THREAD

**Issue by juherr** [cbeust/testng-eclipse#255]
```
java.lang.IllegalArgumentException: invalid version "6.3-201110161418": non-numeric "3-201110161418"
    at org.osgi.framework.Version.parseInt(Version.java:170)
    at org.osgi.framework.Version.<init>(Version.java:130)
    at org.testng.remote.RemoteTestNG.getTestNGVersion(RemoteTestNG.java:87)
    at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:39)
Caused by: java.lang.NumberFormatException: For input string: "3-201110161418"
    at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
    at java.lang.Integer.parseInt(Integer.java:580)
    at java.lang.Integer.parseInt(Integer.java:615)
    at org.osgi.framework.Version.parseInt(Version.java:168)
    ... 3 more
```

From http://stackoverflow.com/q/37668960/4234729

**Comment by cbeust** [comment]
So not really a bug, right? The use just needs to update their TestNG.

-- 
Cédric

**Comment by missedone** [comment]
not a bug, just to show some more meaningful message in case of being failed to detect the testng version on classpath. (Even that I have no any idea where comes the version number `6.3-201110161418`, checked through the release version from the maven central repo ...)

anyway, the user need to update their TestNG for sure.

**Comment by juherr** [comment]
+1 not a bug, but the exception had to be more friendly because some people could have strange environments.

Is it possible to have a popup from the plugin saying that the user has to upgrade its testng dependency because unsupported?

**Comment by missedone** [comment]
hi @juherr 
well, i don't think we can achieve this with simple move.
since fix of #243, the version check was moved to testng-remote, which means it performs the check at TestNG runtime process rather than previously in Eclipse before launch the runtime process.

**Comment by juherr** [comment]
Ok
